packages/connectors-lib/src/sales-api-connector.js

```diff
@@ -27,7 +27,7 @@ export const call = async (url, method = 'get', payload = null) => {
     ok: response.ok,
     status: response.status,
     statusText: response.statusText,
-    body: response.status !== 204 ? await parseResponseBody(response) : undefined
+    body: response.status === 204 ? undefined : await parseResponseBody(response)
```

Comment:
I'm not saying to change this, but simply so you're aware, you could also do:
```
  body: response.status === 204 && await parseResponseBody(response)
```
which would be slightly more succinct, but perhaps not quite as clear in intent